SchoolNormalization.md

```diff
@@ -30,8 +30,12 @@ Example: https://api.careerbuilder.com/core/normalizedschools?query=georgia%20te
             {
                 "normalized_school_name": "Georgia Institute of Technology",
                 "id": "53bff579e4b04710d09fa98d",
+                "confidence": 2.0,
```

Comment:
@msundlof Actually the confidence ranges from 0 to 1 in v2. v1 has a weird scoring system that gives score from 0 to 1 and gives score of 2 when it matches some condition.
We might want to correct this in case it confuses the customer.